sdcm/provision/azure/utils.py

```diff
@@ -71,7 +62,11 @@ def get_scylla_images(  # pylint: disable=too-many-branches,too-many-locals
         for image in gallery_image_versions:
             if image.location != region_name or image.name.startswith('debug-'):
                 continue
-
+            try:
+                int(image.tags.get("build_id"))
+            except (ValueError, TypeError):
+                # skip images with invalid build_id (e.g. created manually)
```

Comment:
Please, add "debug" logging.

Reply:
There's around 170 images from 4.6/4.7/others from year 2021 where Azure was not supported. Do we want these to pollute logs?

Reply:
No, there's no point in that.